### DISCUSSION TITLE
Is the project alive or it was superseded by another create?

### DISCUSSION THREAD

**Discussion by mispecto** [zip-rs/zip-old#435] [Q&A]
Hi,

The last commit was about 5 months ago, and there are pretty critical issues, like #433.
Which makes me wonder if the project is abandoned or not yet :)

Thanks!

**Comment by Sytten** [comment]
@Plecra You might want to put a disclaimer on the README again, since it does look bit unmaintained with 45 PR and last release 10 months ago.
